modules/ols-creating-the-credentials-secret-using-cli.adoc

```diff
@@ -17,9 +18,13 @@ Create a file that is associated with the secret key used to access the API of y
 
 . Create a file that contains the following YAML content:
 +
-.Example credential secret for LLM provider
+[NOTE]
+====
+The YAML parameter is always `apitoken` regardless of what the LLM provider calls the access details.
+====
 +
-[source,yaml, subs="attributes,verbatim"]
+.Example credential secret for LLM provider
```

Comment:
Perhaps out of scope but I don't think that this specific YAML snippet is technically an "example" if it's not fully filled out with example values, in this case there are user-replaced values. You don't have to resolve this in this exact PR, but maybe the word example should be removed from this specific title (the others are examples for sure)
```suggestion
.Credential secret for LLM provider
```

Reply:
done